README.md

```diff
@@ -91,8 +91,6 @@ Wraps all the content that will be conditionally shown when the step is active.
 #### Props
 ##### `path`: string
 Unique key for each step.
-##### `name`: string *(optional)*
```

Comment:
`name` is still supported but is treated the same as any other custom config prop that is added to `<Step />`.  I don't think we should list any specific custom prop in the docs, but maybe it would be helpful to have a section saying:

> In addition to `path` and `className`, any additional props provided to `<Step>` will be available on the `step` object.